index.js

```diff
@@ -8,9 +8,11 @@ module.exports = {
     var config = this._super.config.apply(this, arguments);
 
     var version = require('git-repo-version')(null, this.project.root);
-    if (version && baseConfig.APP) {
+    let gitCommand = "git log --max-count=1 --format='%aI'";
```

Comment:
Instead of calling `git` here, we should use `authorDate` from [git-repo-info](https://github.com/rwjblue/git-repo-info#usage). 

You will need to create a PR to [git-repo-version](https://github.com/cibernox/git-repo-version/blob/master/index.js#L11) with this new feature. Once the PR is accepted, circle back to this PR.